Source/GSSocketStream.m

```diff
@@ -1893,11 +1893,11 @@ - (id) propertyForKey: (NSString *)key
 {
   id	result = [super propertyForKey: key];
 
-  if (result == nil && _address.s.sa_family != AF_UNSPEC)
+  if (result == nil && _address.s.ss_family != AF_UNSPEC)
     {
-      SOCKET    	s = [self _sock];
       struct sockaddr	sin;
-      socklen_t	        size = sizeof(sin);
+      SOCKET    	s = [self _sock];
+      socklen_t	  size = sizeof(sin);
```

Comment:
something doesn't seem right here. sockaddr may be too small do you think sin should actually be of type sockaddr_storage?

Reply:
I think this is showing up in the diff because of the spacing, I will make sure to revert this spacing change.

Reply:
Yea i think we will leave it alone for now. I can see how its definitely a bug but I can't see how we fix it easily in this MR. We will leave it alone for now.